### ISSUE TITLE
How long did you complete the training process

### ISSUE THREAD

**Issue by zwzhang123** [motokimura/PyTorch_Gaussian_YOLOv3#36]
training on COCO dataset?

**Comment by motokimura** [comment]
Around 3 to 4 weeks (with V100 x 1 or 1080Ti x 1).

**Comment by zwzhang123** [comment]
thx, does not support multi-GPU training?

**Comment by motokimura** [comment]
You're welcome. No, not supported currently..

**Comment by zwzhang123** [comment]
oh, i am  writing the codes for multi-GPU training. Has the loss converged on 500000 steps training?

**Comment by sisrfeng** [comment]
Mutil gpu : please refer to https://github.com/DeNA/PyTorch_YOLOv3/issues/23

**Comment by zwzhang123** [comment]
> Mutil gpu : please refer to DeNA#23

thx, but i want to use model = torch.nn.parallel.DistributedDataParallel(model). and some bugs are being fixed.

**Comment by sisrfeng** [comment]
> > Mutil gpu : please refer to [DeNA#23](https://github.com/DeNA/PyTorch_YOLOv3/issues/23)
> 
> thx, but i want to use model = torch.nn.parallel.DistributedDataParallel(model). and some bugs are being fixed.

would you mind sharing your code later?

**Comment by zwzhang123** [comment]
en, my pleasure.

**Comment by motokimura** [comment]
@Dreamerzzw,

> oh, i am writing the codes for multi-GPU training. Has the loss converged on 500000 steps training?

Yes. See [here](https://github.com/motokimura/PyTorch_Gaussian_YOLOv3#performance).
